src/Resources/js/modules/combined-meal-service.ts

```diff
@@ -0,0 +1,109 @@
+export class CombinedMealService {
+    public static getCombinedMealDishes($meal: JQuery): Dish[] {
```

Comment:
May be we can rename $meal to $mealContainer to be more clear.

Reply:
Sure! Good point